srcpkgs/python3-pmw/template

```diff
@@ -0,0 +1,19 @@
+# Template for python-pmw
+pkgname=python3-pmw
+version=2.0.1
+revision=1
+noarch=yes
+short_desc="Python 2 Tkinter widget toolkit"
```

Comment:
No Python 3?

Reply:
Ah yes, I split it into two and then I forgot this little change when I went back to just 3.